easybuild/easyconfigs/b/BamUtil/BamUtil-1.0.15-GCC-12.2.0

```diff
@@ -0,0 +1,38 @@
+name = 'BamUtil'
+version = '1.0.15'
+
+homepage = 'http://genome.sph.umich.edu/wiki/BamUtil'
+description = """BamUtil is a repository that contains several programs
+  that perform operations on SAM/BAM files. All of these programs
+  are built into a single executable, bam."""
+
+toolchain = {'name': 'GCC', 'version': '12.2.0'}
+
+sources = [
+    {
+        'source_urls': ['https://github.com/statgen/BamUtil/archive/'],
+        'download_filename': 'v%(version)s.tar.gz',
+        'filename': 'BamUtil-%(version)s.tar.gz',
+    },
+    { 
+        'source_urls': ['https://github.com/statgen/libStatGen/archive/'],
+        'download_filename': 'v%(version)s.tar.gz',
+        'filename': 'libStatGen-%(version)s.tar.gz',
+    },
+]
+
+dependencies = [('zlib', '1.2.11')]
+
+# Ensure the right compilers are used
+buildopts = 'CC="$CC" CXX="$CXX" USER_WARNINGS="" LIB_PATH_BAM_UTIL=%(builddir)s/libStatGen-*/'
+
+files_to_copy = ['bin']
+
+sanity_check_paths = {
+    'files': ['bin/bam'],
+    'dirs': [],
+}
+
+easyblock = 'MakeCp'
```

Comment:
please move the easyblock parameter to the top of the easyconfig. we are quite strict in the order of parameters for contributions.